addon/components/frost-fixed-table.js

```diff
@@ -278,6 +329,32 @@ export default Component.extend({
     this.$(`${bodyMiddleSelector} .frost-table-row`).css({width: `${width}px`})
   },
 
+  setupLeftAndRightWidths () {
+    const sides = [
+      {
+        header: this.get('headerLeftSelector'),
+        body: this.get('bodyLeftSelector')
+      }, {
+        header: this.get('headerRightSelector'),
+        body: this.get('bodyRightSelector')
+      }
+    ]
+    for (let side of sides) {
+      const headerCells = this.$(`${side.header} .frost-table-header-cell`)
+      for (let pos = 0; pos < headerCells.length; ++pos) {
```

Comment:
Any reason to use `++pos` instead of `pos++` 

This change will avoid doing a `pos + 1`